client/post-editor/editor-ground-control/index.jsx

```diff
@@ -187,6 +187,18 @@ export class EditorGroundControl extends PureComponent {
 		return this.props.translate( 'Saved' );
 	}
 
+	getE2ESaveStatus() {
+		if ( this.props.isSaving ) {
```

Comment:
Optional - we could destructure here to avoid repeating `this.props`:
`const { isSaving, post } = this.props;`